File: backend/DataAccess/DaoGenerated/システム共有/tt_afsiyorirekiDto.cs
// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: DTO定義
//             使用履歴テーブル
// 作成日　　: 2023.01.23
// 作成者　　: 屠
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tt_afsiyorirekiDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tt_afsiyorireki";
		public string userid { get; set; }                        //ユーザーID
		public string kinoid { get; set; }                        //機能ID
		public DateTime syoridttm { get; set; }                   //処理日時
    }
}

File: backend/JbdGjsService/Db/DaoGenerated/システム共有/tt_afsiyorirekiDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             使用履歴テーブル
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace JBD.GJS.Db
    Public Class tt_afsiyorirekiDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tt_afsiyorireki"
        Public Property userid As String                        'ユーザーID
        Public Property kinoid As String                        '機能ID
        Public Property syoridttm As Date                   '処理日時
    End Class
End Namespace
